lib/mysql2/client.rb

```diff
@@ -105,9 +105,12 @@ def info
       self.class.info
     end
 
-    private
-      def self.local_offset
+    class << self
+      private
+
+      def local_offset
```

Comment:
I've always prefered the `def self.foo` syntax, is there a reason that RuboCop prefers the class << self block syntax?